src/templates/index.html

```diff
@@ -14,8 +14,10 @@
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     <link rel="stylesheet" href="https://storage.googleapis.com/code.getmdl.io/1.0.0/material.min.css">
     <link rel="stylesheet" href="styles.css">
-    <style>
-    #view-source {
+    
+      <style type="text/css">
```

Comment:
CSS styles need to live separate from the index page, and should not be embedded in a `style` tag in the html